agent/tool-scripts/unittests

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
-export LANG=C
-export LC_ALL=C
```

Comment:
Why were these removed?  We had to add them in the past in order to make sure the test output was correct across multiple platforms.

Reply:
Doh! Fixed